System.Maui.ControlGallery.Android/Tests/TestingPlatformService.cs

```diff
@@ -11,9 +10,11 @@ class TestingPlatformService : ITestingPlatformService
 	{
 		public async Task CreateRenderer(VisualElement visualElement)
 		{
-			await Device.InvokeOnMainThreadAsync(() => 
-				Platform.Android.Platform.CreateRendererWithContext(visualElement,
-					DependencyService.Resolve<Context>()));
+			//await Device.InvokeOnMainThreadAsync(() =>
+			//	Platform.Android.AppCompat.Platform.CreateRendererWithContext(visualElement,
```

Comment:
Pre-AppCompat Platform was public, now it's internal, not sure how to resolve this